tests/components/bsblan/test_config_flow.py

```diff
@@ -92,3 +118,42 @@ async def test_full_user_flow_implementation(
 
     entries = hass.config_entries.async_entries(config_flow.DOMAIN)
     assert entries[0].unique_id == "RVS21.831F/127"
+
+
+async def test_full_user_flow_implementation_without_auth(
+    hass: HomeAssistant, aioclient_mock
+) -> None:
+    """Test the full manual user flow from start to finish."""
+    aioclient_mock.post(
+        "http://example2.local:80/JQ?Parameter=6224,6225,6226",
+        text=load_fixture("bsblan/info.json"),
+        headers={"Content-Type": CONTENT_TYPE_JSON},
+    )
+
+    result = await hass.config_entries.flow.async_init(
+        config_flow.DOMAIN,
+        context={"source": SOURCE_USER},
+    )
+
+    assert result["step_id"] == "user"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+
+    result = await hass.config_entries.flow.async_configure(
```

Comment:
We normally patch the integration setup functions on all create entry results to avoid setting up the whole integration. See the scaffold script test template for example.

Reply:
Thanks Martin I'll have a look at the scaffold

Reply:
Can I save this adjustment for the next update? I see that the scaffold has changed a lot and it will take me some time to adjust this.